js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -248,8 +257,8 @@ export const tensorTypeToWsglValueType = (type: DataType, components: 1|2|3|4 =
  *    vec4.
  */
 const createIndicesHelper =
-    (name: string, tensorType: number, shape: readonly number[], isInput: boolean,
-     components: 1|2|3|4): IndicesHelper => {
+    (name: string, tensorType: number, shape: readonly number[], isInput: boolean, components: 1|2|3|4,
+     output?: IndicesHelper): IndicesHelper => {
```

Comment:
I suggest to put this `output` as parameter of `broadcastIndicesToOffset` instead.